### ISSUE TITLE
[Geometry] CV Basement vertical grey lines (cobweb related?)

### ISSUE THREAD

**Issue by sicoderealm** [Meridian59/Meridian59#540]
The CV Basement has a few vertical lines on the stairs and around the boxes - possibly related to the cobwebs?

<img width="1293" alt="image" src="https://github.com/Meridian59/Meridian59/assets/7548210/1daa4298-5a68-417d-8a24-0dcf45d6dc29">

![image](https://github.com/Meridian59/Meridian59/assets/7548210/5d548eef-6166-4bb5-a220-47cd768f1ae9)

![image](https://github.com/Meridian59/Meridian59/assets/7548210/ac85719c-ab87-4866-b24a-cbf7b4c624aa)

![image](https://github.com/Meridian59/Meridian59/assets/7548210/1c3ebb24-44d3-4ff0-9573-e62044937aca)

and so on...

**Comment by davidklaw** [comment]
So, this can be fixed by adjusting the offset of these textures by 1. However, doing so doesn't feel like the appropriate fix knowing that there are many more instances of this in this room and others. We've seen signs in Jasper fixed in the past, and the gates of Tos and Barloque also have this issue. I'm wondering if there's an adjustment that needs to be made in the code for texture drawing as it seems like a bit of an issue in many place single floating line textures are used. Perhaps everywhere they're used, but most notable when the texture's edge has artwork.

![image](https://github.com/Meridian59/Meridian59/assets/467443/9507b6a6-ac28-45d3-9e9c-e35f50fb9285)

**Comment by akirmse** [comment]
Yeah, agreed.  In rendering there's the concept of wrapping versus clamping
textures.  I wonder if this wall isn't set correctly, or if the hardware
renderer itself isn't setting the mode right.

**Comment by davidklaw** [comment]
```
Seems like it might be tied to this function in `d3drender.c`

Bool D3DMaterialWorldPool(d3d_render_pool_new *pPool)
{
	D3DRENDER_SET_COLOR_STAGE(gpD3DDevice, 0, D3DTOP_MODULATE, D3DTA_TEXTURE, D3DTA_DIFFUSE);
	D3DRENDER_SET_ALPHA_STAGE(gpD3DDevice, 0, D3DTOP_MODULATE, D3DTA_TEXTURE, D3DTA_DIFFUSE);

	IDirect3DDevice9_SetSamplerState(gpD3DDevice, 0,
                                         D3DSAMP_ADDRESSU, D3DTADDRESS_WRAP);
	IDirect3DDevice9_SetSamplerState(gpD3DDevice, 0,
                                         D3DSAMP_ADDRESSV, D3DTADDRESS_WRAP);

	return TRUE;
}
```

When I change `WRAP` to `BORDER` the web appears correctly, but the rest of the world is broken.

![image](https://github.com/Meridian59/Meridian59/assets/467443/7c1a7d04-fee9-4e76-a1da-4bf0c9c43d62)

I'm not sure what to do with this information yet 😆

**Comment by Meridian59** [comment]
This might be tough to fix without adding wrap vs. clamp flags to the
editor.  It looks like there's room to add new wall flags:
https://github.com/Meridian59/Meridian59/blob/165377abe71acbff7bf2ec7f63a433cf86a616aa/clientd3d/bsp.h#L15

**Comment by davidklaw** [comment]
Fixed via #778
